index.js

```diff
@@ -29,6 +30,10 @@ WebpackCopyAfterBuildPlugin.prototype.apply = function(compiler) {
         var from = outputPath + "/" + chunkFilename;
         var to = outputPath + "/" + mapping;
 
+        if( options.absoluteMappingPaths === true ){
```

Comment:
How do you feel about using truthyness here and only assigning the other `to` in the `else` condition? i.e.

```javascript
var to;

if (options.absoluteMappingPaths) {
   to = mapping;
} else {
   to = outputPath + "/" + mapping;
}
```